source/renderer/app/api/api.js

```diff
@@ -41,6 +41,8 @@ import { deleteLegacyTransaction } from './transactions/requests/deleteLegacyTra
 import { selectCoins } from './transactions/requests/selectCoins';
 import { createExternalTransaction } from './transactions/requests/createExternalTransaction';
 import { getPublicKey } from './transactions/requests/getPublicKey';
+import { getICOPublicKey } from './transactions/requests/getICOPublicKey';
+import type { ICOPublicKeyParams } from './transactions/requests/getICOPublicKey';
```

Comment:
Please, move it to the bottom, along with the other transaction types (line 144)

Reply:
Also, please move this type into `api/transactions/types.js`